University-advisor/Forms/MainForm.cs

```diff
@@ -15,9 +15,12 @@ namespace University_advisor.Forms
     public partial class MainForm : Form
     {
         string currentUser;
+        ArrayList statusList = new ArrayList() { "Student", "Graduate", "Lecturer" };
```

Comment:
Use implicitly typed variables, so var statusList.

Reply:
Why is that better? You used ArrayList in SignUpForm

Reply:
Then it needs to be changed :)

Reply:
var can only be used when a **local** variable is declared and initialized in the same statement